compose.go

```diff
@@ -76,6 +80,135 @@ func (p *Plugin) CobraAddCommands(rootCmd *cobra.Command) error {
 	composeCmd.Flags().BoolVar(&conflictsVerbosity, "conflicts-verbosity", false, "Log files conflicts")
 	composeCmd.Flags().BoolVar(&clean, "clean", false, "Remove .compose dir on start")
 	composeCmd.Flags().BoolVar(&interactive, "interactive", true, "Interactive mode allows to submit user credentials during action")
+
+	composeDependency := &compose.Dependency{}
+	strategies := &compose.RawStrategies{}
+	var createNew bool
+	var addCmd = &cobra.Command{
+		Use:   "compose:add",
```

Comment:
Would be great to sanitize fields input by removing trailing and leading spaces (or any?)

![image](https://github.com/launchrctl/compose/assets/20738264/4ea01ae2-2a92-4123-bb58-0a0a293209ad)

![image](https://github.com/launchrctl/compose/assets/20738264/2c2a8be7-a4ae-4032-9323-17e7a0468309)